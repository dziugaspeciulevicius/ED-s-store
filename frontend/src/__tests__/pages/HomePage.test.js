import React from "react";
import { shallow } from "enzyme";
import "../../setupTests";
import HomePage from "../../pages/HomePage.component"
jest.mock("../../pages/HomePage.component.jsx");

describe("<HomePage />", () => {
    it("Renders <HomePage /> component correctly", () => {
      // shallow creates instance of a component
      expect(shallow(<HomePage />)).toMatchSnapshot();
    });
});
