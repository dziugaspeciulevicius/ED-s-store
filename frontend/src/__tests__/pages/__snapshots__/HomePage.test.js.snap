// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HomePage /> Renders <HomePage /> component correctly 1`] = `ShallowWrapper {}`;
